hack/release.toml

```diff
@@ -99,6 +99,15 @@ Talos can be configued to use Kubernetes 1.21 or CAPI v0.4.x components can be u
         description = """\
 Added support for Equinix Metal IPs for the Talos virtual (shared) IP (option `equnixMetal` under `vip` in the machine configuration).
 Talos automatically re-assigns IP using the Equinix Metal API when leadership changes.
+"""
+
+    [notes.configuration]
+        title = "Machine Config Validation"
+        description = """\
+Unknown keys in the machine config now make the config invalid, 
+so any attempt to apply/edit the configuration with the unknown keys will lead into an error.
+As a consequence of this change any downgrades to version 0.12+ may need to be preceeded with configuration file update,
```

Comment:
I would remove this sentence, as it's too complicated to understand and not really for 0.12, as there's no problem downgrading to 0.11. I would rather save it for 0.13 if that would even be a problem (ever)